### PR TITLE
Check for `undefined` body

### LINKED ISSUE #63: GET requests have data as string "undefined"
I'm running into an issue where a GET request is failing because the `axios.request(config)` is being called with an invalid `data` property:

```
// config object
{
  ...
  "method": "GET",
  "data": "undefined",
  ...
}
```

This is due to `config.data` being set [here](https://github.com/lifeomic/axios-fetch/blob/86b4b700b711167b093d43123ec79f98eccf697b/src/index.js#L24) to `String(init.body)` when `init.body === undefined`. 

I'll submit a PR for review.

### PR DESCRIPTION
**Pull request by hogpilot**
Checks if `init.body` is defined before converting it to a string in `axiosFetch` method.

Fixes #63

### CODE REVIEW

**Comment by mdlavin** [comment]
@hogpilot Thank you for contributing a fix and your change looks good to me. Would you mind adding a testcase to make sure that it doesn't get regressed by future changes?

**Review by mdlavin — APPROVED** [review]

**Comment by mdlavin** [comment]
@hogpilot Double thanks for the fix and test. I'll wait on the update on the PR and will release a new version after that one merges too

**Comment by hogpilot** [comment]
@mdlavin Thanks for your quick review.  I appreciate the test reminder

**Comment by mdlavin** [comment]
This fixe long with #62 should be included in @lifeomic/axios-fetch@1.5.2

**Pull request merged**
